packages/buffetjs-custom/src/components/Inputs/Wrapper.js

```diff
@@ -10,14 +10,14 @@ import { colors, sizes } from '@buffetjs/styles';
 const Wrapper = styled.div`
   position: relative;
   padding-bottom: ${sizes.margin * 2.7}px;
-  margin-bottom: ${sizes.margin * 1.7}px;
+  margin-bottom: ${sizes.margin * 1.8}px;
   label {
     display: block;
-    margin-bottom: 1.1rem;
+    margin-bottom: 1rem;
   }
   p {
     position: absolute;
-    bottom: 0;
+    bottom: -1px;
```

Comment:
Can you please let `bottom: 0;`, but add the `line-height: 1.6rem;` attribute instead?